src/components/CompanyAndJobTitle/Overview/EmployerAggregateRating.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Seo from 'common/Seo/SeoStructure';
+import PropTypes from 'prop-types';
+import { ORIGIN } from 'config';
+
+const EmployerAggregateRating = ({
+  title,
+  description,
+  companyName,
+  averageRating,
+  ratingCount,
+}) => {
+  return (
+    <Seo
+      data={{
+        '@context': 'https://schema.org/',
+        '@type': 'EmployerAggregateRating',
+        name: title,
+        description,
+        itemReviewed: {
+          '@type': 'Organization',
+          name: companyName,
+          sameAs: ORIGIN,
+        },
+        ratingValue: averageRating.toFixed(1),
```

Comment:
should this be number ?

this PR: `"ratingValue":"3.9","ratingCount":3248`

interview dot tw
```
"ratingValue":3.8,"bestRating":5,"worstRating":1,"ratingCount":264
```